base/compiler/inferencestate.jl

```diff
@@ -242,6 +251,7 @@ end
 
 # temporarily accumulate our edges to later add as backedges in the callee
 function add_backedge!(li::MethodInstance, caller::InferenceState)
+    caller.linfo !== nothing || return # don't add backedges to opauqe closures
```

Comment:
```suggestion
    caller.linfo !== nothing || return # don't add backedges to opaque closures
```

Reply:
Should this be a TODO XXX? I'd assume that OpaqueClosures do have edges (ie. can call functions), so we need to be able to propagate those back to their caller